x-pack/plugins/observability_solution/metrics_data_access/common/inventory_models/container/metrics/charts/index.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { cpu } from './cpu';
+import { memory } from './memory';
+
+export const charts = {
+  cpu,
+  memory,
+} as const;
+
+export type ContainerCharts = typeof charts;
```

Comment:
How will we extend it when we start supporting more metrics for Docker container?

Reply:
I will do it as part of the follow-up